layouts/index.html

```diff
@@ -96,7 +93,34 @@ <h1 class="display-5 fw-bold lh-1 mb-5 mb-md-3">
             </div>
           </div>
         </div>
-
+        <!-- USED BY -->
+		<div class="al-index-used-by al-py-lg" >
+            <div class="container">
+                <h4 class="pb-2 border-bottom">
+                    {{ i18n "Trusted by" }}
+                </h4>
```

Comment:
Consistency is very good, for sure, and your suggestion is correct if we want to keep it.

As a note: I'm worried about how much space this title in addition to the logos adds to the page, though, and was considering dropping the text entirely here. That seemed consistent with what other organizations do when providing the same information (or maybe even change it to something a bit more verbose 'Our users come from industries across the globe'). 

However, I'm worried that it won't be clear what these logos represent without the text. What do you think?

Reply:
Ah! In this case, how about placing this info and logos right under the very first and main section? It will take less space and can work even without sections colors adjustments
![image](https://github.com/AlmaLinux/almalinux.org/assets/87830789/4b207315-85fa-4387-8315-55fc2dbcb9dc)

Reply:
OH! I like that a TON, actually.

Reply:
Adding this should do the job

```
<div class="al-index-backed-by al-py-lg" style="width: 100%; padding: 0 15px;">
                    <div class="container">
                        <h3 class="pb-2">
                            Trusted by
                        </h3>
                        <div class="row row-cols-2 row-cols-sm-4 row-cols-md-6 g-5 al-used-by-list ">
                                
                                    <div class="col d-flex flex-fill align-items-center justify-content-center al-used-by-logo-container">
                                        <img loading="lazy" src="/usedby/akamai.svg" alt="akamai" class="al-used-by-logo" style="height: 100%; width: 120px;">
                                    </div>        
                            
                                    <div class="col d-flex flex-fill align-items-center justify-content-center al-used-by-logo-container">
                                        <img loading="lazy" src="/usedby/arista.svg" alt="arista" class="al-used-by-logo" style="height: 100%; width: 120px;">
                                    </div>
                            
                                    <div class="col d-flex flex-fill align-items-center justify-content-center al-used-by-logo-container">
                                        <img loading="lazy" src="/usedby/colorodostateuniveristy.svg" alt="colorodostateuniveristy" class="al-used-by-logo" style="height: 100%; width: 120px;">
                                    </div>

                                    <div class="col d-flex flex-fill align-items-center justify-content-center al-used-by-logo-container">
                                        <img loading="lazy" src="/usedby/hetzner.png" alt="hetzner" class="al-used-by-logo" style="height: 100%; width: 120px;">
                                    </div>
                                    
                                    <div class="col d-flex flex-fill align-items-center justify-content-center al-used-by-logo-container">
                                        <img loading="lazy" src="/usedby/lacework.svg" alt="lacework" class="al-used-by-logo" style="height: 100%; width: 120px;">
                                    </div>
                                    
                                    <div class="col d-flex flex-fill align-items-center justify-content-center al-used-by-logo-container">
                                        <img loading="lazy" src="/usedby/phoronix.svg" alt="phoronix" class="al-used-by-logo" style="height: 100%; width: 120px;">
                                    </div>
                        </div>
                    </div>
                </div>
                        
```